### ISSUE TITLE
Chef is grep'ing for HBase services

### ISSUE THREAD

**Issue by cbaenziger** [bloomberg/chef-bach#53]
It appears for our [hbase-master service](https://github.com/bloomberg/chef-bach/blob/b7bdfeefe42b52c58a5a15e77ded2d3298da46c2/cookbooks/bcpc-hadoop/recipes/hbase_master.rb#L33-L38) is falling back to walking the process list for the regular expression `/hbase-master/` which means should a user be running a processes like:

````
user1 14724 13749  0 Dec16 pts/4    00:00:00 tail -f hbase-hbase-master-cluster1-r1n8.log
````

Then the Chef run will frustratingly not start the service should it be down; running Chef with debug logging yields:
````
[2014-12-17T17:57:52-05:00] INFO: Processing service[hbase-master] action enable (bcpc-hadoop::hbase_master line 68)
[2014-12-17T17:57:52-05:00] DEBUG: Platform ubuntu version 12.04 found
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] falling back to process table inspection
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] attempting to match 'hbase-master' (/hbase-master/) against process list
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] running: true
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 0, action stop, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 1, action stop, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 2, action start, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 3, action start, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 4, action start, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 5, action start, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 6, action stop, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] already enabled - nothing to do
[2014-12-17T17:57:52-05:00] INFO: Processing service[hbase-master] action start (bcpc-hadoop::hbase_master line 68)
[2014-12-17T17:57:52-05:00] DEBUG: Platform ubuntu version 12.04 found
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] falling back to process table inspection
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] attempting to match 'hbase-master' (/hbase-master/) against process list
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] running: true
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 0, action stop, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 1, action stop, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 2, action start, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 3, action start, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 4, action start, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 5, action start, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] runlevel 6, action stop, priority 20
[2014-12-17T17:57:52-05:00] DEBUG: service[hbase-master] already running - nothing to do
````

We need to set `supports :restart => false, :reload => false, :status => false`; perhaps we can even set the default for our cookbook?

**Comment by bijugs** [comment]
Fixed as part of PR #110. Can we close this?

**Comment by bijugs** [comment]
Fixed in PR #110. Thanks @cbaenziger .
